DependencyInjection/EkoFeedExtension.php

```diff
@@ -60,6 +60,6 @@ protected function configureHydrator(array $config, ContainerBuilder $container)
         }
 
         $container->getDefinition('eko_feed.feed.reader')
-            ->setArguments(array($config['hydrator']));
+            ->addMethodCall('setHydrator', array(new \Symfony\Component\DependencyInjection\Reference($config['hydrator'])));
```

Comment:
Can you please use:

use Symfony\Component\DependencyInjection\Reference

and new Reference(...) here please?

Thank you!